src/main/java/com/thevoxelbox/voxelsniper/brush/DrainBrush.java

```diff
@@ -33,7 +33,7 @@ private void drain(final SnipeData v) {
         final int brushSize = v.getBrushSize();
         final double brushSizeSquared = Math.pow(brushSize + this.trueCircle, 2);
         final Undo undo = new Undo();
-
+        //TODO should y be clamped to 0?
```

Comment:
Would need a closer look I think

Reply:
Indeed, that's why I made a todo. I'll make it an issue though

Reply:
was about to do it